fetcher/prices.go

```diff
@@ -40,6 +40,11 @@ func (provider *PriceProvider) PrimaryIP(ipType hcloud.PrimaryIPType, datacenter
 	provider.pricingLock.RLock()
 	defer provider.pricingLock.RUnlock()
 
+	// v6 pricing is not defined by the API
+	if string(ipType) == "ipv6" {
+		return 0, 0, nil
+	}
+
 	for _, byType := range provider.getPricing().PrimaryIPs {
 		if byType.Type == string(ipType) {
 			for _, pricing := range byType.Pricings {
```

Comment:
After I looked at your PR on the hcloud-go repository, this makes sense, but I am actually quit suprised that the `Datacenter` attribute is now actually the location. To me, this is counter-inuitive. I would at least refactor this method to consistently call it location and add a comment why its then set to `p.Datacenter` 🙂.